browser/admin/src/AdminSocketOverview.js

```diff
@@ -379,6 +386,15 @@ var AdminSocketOverview = AdminSocketBase.extend({
 			var $mod = $(document.getElementById('mod' + sPid));
 			$mod.text(value);
 		}
+		else if (textMsg.startsWith('uploaded')) {
+			textMsg = textMsg.substring('uploaded'.length);
+			docProps = textMsg.trim().split(' ');
+			sPid = docProps[0];
+			var value = docProps[1];
+
+			var $up = $(document.getElementById('up' + sPid));
+			$up.text(value);
```

Comment:
Surely we should check that `$up !== undefined`. Also, the `$` seems a typo, as we don't use '$' in variable names.